js/app.js

```diff
@@ -14,34 +14,30 @@ function webcam() {
   };
 
   navigator.mediaDevices.getUserMedia(constraints)
-    .then(function (mediaStream) {
-      var video = document.querySelector('video');
+    .then(function(mediaStream) {
+      var video = document.querySelector("video"); // is this supposed to be "#video"?
```

Comment:
I think this was meant to select the video tags, not the ID. However, I commented it out and it works without it. I guess it's because the next lines actually does select the video tags. So probably delete it?